src/components/badge/examples/badge-number.tsx

```diff
@@ -1,8 +1,14 @@
 import { Component, h } from '@stencil/core';
 
+/**
+ * Number badges
+ *
+ * Numbers containing more than 3 digits get abbreviated
+ * using number units (K, M, B, and T).
+ */
 @Component({
-    tag: 'limel-example-badge',
-    styleUrl: 'badge.scss',
+    tag: 'limel-example-badge-number',
```

Comment:
![image](https://user-images.githubusercontent.com/35954987/194822692-83345504-24e5-4b25-9fdc-ef7627bf49ab.png)
The reason for this is that you renamed the component tag here. But the CSS is still saying `:host(limel-example-badge) {` So you should just add a -badge in there too 😊